shub/log.py

```diff
@@ -36,13 +36,33 @@
 SHORT_HELP = "Fetch log from Scrapy Cloud"
 
 
+class LogIterFunc(object):
```

Comment:
This looks like a big class just for setting an initial `startafter`. Maybe we can move the "Determine first `_startafter`" logic into `cli()` and allow a `startafter` keyword arg to `job_resource_iter`?